lib/serverengine/daemon.rb

```diff
@@ -47,14 +47,23 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @chuser = @config[:chuser]
       @chgroup = @config[:chgroup]
       @chumask = @config[:chumask]
+
+      @pid = nil
+      @command_pipe = @config.fetch(:command_pipe, nil)
```

Comment:
Even if any values are passed from `Daemon.run_server` into `@command_pipe`, `@command_pipe` seems to be updated without any consideration about existing values at L142.
https://github.com/unak/serverengine/blob/7b485e72d58ac77f04132b4efef81e55fc22d33c/lib/serverengine/daemon.rb#L142

Are there any other place which existing `@command_pipe` values are used?